modules/scratchFetch.ts

```diff
@@ -0,0 +1,242 @@
+import { defineEvent } from "strife.js";
+import { getSettings } from "./settings.js";
+import constants from "../common/constants.js";
+
+defineEvent("messageCreate", async (message) => {
+	const notSet = getSettings(message.author, false)?.scratchEmbeds === undefined;
```

Comment:
settings need to be coded again, honestly not sure how you're getting successful builds